Update UG and DG with regards to PE Dry Run (#266)
AY2425S1-CS2103T-F14b-3/tp: docs/UserGuide.md

* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.

* The `personList` view is the view where the list of elderly is shown. It is the default view when you start the application. Use the `list` command to return to this view if you are in another view.

> * The fields in a command cannot have their prefix as part of the input. For example, `address` cannot be `a/Clementi Ave a/2` or `notes` cannot be `o/This is a note o/part of same note`.

Reviewer: Not sure if this sounds better to account for `o/This is a noteo/part of same note` which should work?
```suggestion
* The fields in a command cannot contain whitespace followed by a prefix within the input. For example, `address` cannot be `a/Clementi Ave a/2` or `notes` cannot be `o/This is a note o/part of same note`.
```

Reviewer 2: I think can clarify and give an example that its not just their prefix, but also other prefixes of that the command can accept.

Like for eg. you can't do `mark 1 o/This is a note d/part of same note`
